build.sbt

```diff
@@ -46,7 +46,7 @@ lazy val commonDependencies = Seq(
 lazy val buildSettings = Seq(
   organization  :=  "com.snowplowanalytics",
   name          :=  "snowplow-stream-collector",
-  version       :=  "1.0.1",
+  version       :=  "2.0.0",
```

Comment:
I think this change should be part of the `Prepare for release` commit. We used to have it as a separate step in the umbrella releases, because it was possible to release a new `R11x` version of `snowplow` and *not* bump the collector. But now that it's in its own repo, we should follow the practice from all our others single-project repos.

Reply:
Agreed with Dilyan.

Reply:
done